pkg/v1/tkg/web/src/app/shared/components/import-file-success-notification/import-file-success-notification.component.html

```diff
@@ -0,0 +1,15 @@
+<div *ngIf="successImportFile" class="alert alert-success" role="alert">
```

Comment:
Is there a chance that you are not rebased with main? The reason that I ask is because I believe we removed this component to use a shared alert-notification component.